### PR TITLE
VSEE - Added page for Prisma Scanner integration

### PR DESCRIPTION
**Pull request by chaomonica**
Which other branches do you want a technical writer to cherry-pick this PR to (if any)? none

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: scst-scan/install-prisma-integration.hbs.md [hunk 1]
```diff
@@ -0,0 +1,566 @@
...
+
+## Prequisites for Prisma Scanner (Beta)
+
+This document describes prerequisites for installing Supply Chain Security Tools - Scan (Prisma) from the VMware package repository.
+
+### Prepare the Prisma Scanner configuration
+
+To prepare the Prisma configuration before you install any scanners:
+
+1. Obtain a Prisma Token from Prisma.
```
Comment (rxl7906) [inline scst-scan/install-prisma-integration.hbs.md:120]: Is there a link for this?

File: scst-scan/install-prisma-integration.hbs.md [hunk 2]
```diff
@@ -0,0 +1,566 @@
...
+```
+
+**Without Supply Chain Security Tools - Store Integration:** If you don’t want to enable the Supply Chain Security Tools - Store integration, explicitly deactivate the integration by appending the following fields to the values.yaml file that is enabled by default:
+
+```
+# ...
+metadataStore:
+  url: "" # Configuration is moved, so set this string to empty
+```
+### Sample ScanPolicy for CycloneDX Format
```
Comment (rxl7906) [inline scst-scan/install-prisma-integration.hbs.md:224]: For some reason the format on the actual copy renders the following outside the code block:
```
apiVersion: scanning.apps.tanzu.vmware.com/v1beta1 kind: ScanPolicy metadata: name: scan-policy labels: 'app.kubernetes.io/part-of': 'enable-in-gui' spec: regoFile: | package main```

To fix: put a opening ``` and closing ``` for this scanpolicy code block

File: scst-scan/install-prisma-integration.hbs.md [hunk 3]
```diff
@@ -0,0 +1,566 @@
...
+For example:
+
+```
+$ tanzu package available list prisma.scanning.apps.tanzu.vmware.com --namespace tap-install
+/ Retrieving package versions for prisma.scanning.apps.tanzu.vmware.com...
+  NAME                                  VERSION           RELEASED-AT
+  prisma.scanning.apps.tanzu.vmware.com   0.1.0-beta.4
+```
+
+3. (Optional) Create the secrets the scanner package relies on:
```
Comment (rxl7906) [inline scst-scan/install-prisma-integration.hbs.md:322]: Where's the secrets? I don't see it in the original page either.

File: scst-scan/install-prisma-integration.hbs.md (hunk 1 above)
Comment (rxl7906) [reply scst-scan/install-prisma-integration.hbs.md:120]: Not necessary, link could change in the future, and the user would most likely know how to get it already.

File: scst-scan/install-prisma-integration.hbs.md (hunk 2 above)
Comment (chaomonica) [reply scst-scan/install-prisma-integration.hbs.md:224]: done!

**Review by rxl7906 — APPROVED** [review]
LGTM

File: scst-scan/install-prisma-integration.hbs.md (hunk 3 above)
Comment (rxl7906) [reply scst-scan/install-prisma-integration.hbs.md:323]: @Mpluya @x95castle1 Can you address this?

**Pull request merged**